notary-eth-integration-test/build.gradle

```diff
@@ -13,7 +13,7 @@ sourceSets {
             srcDirs += 'src/integration-test/kotlin'
         }
         resources {
-            srcDirs = ["src/integration-test/resources"]
+            srcDirs += ["src/integration-test/resources"]
```

Comment:
Do we have to fix it in other repositories?

Reply:
I think only if we have problems.